src/snowflake/snowpark/modin/plugin/extensions/series_overrides.py

```diff
@@ -164,3 +164,29 @@ def plot(
 @series_not_implemented()
 def transform(self, func, axis=0, *args, **kwargs):  # noqa: PR01, RT01, D200
     pass  # pragma: no cover
+
+
+@register_series_accessor("is_monotonic_increasing")
+@property
+@snowpark_pandas_telemetry_method_decorator
+def is_monotonic_increasing(self):  # noqa: RT01, D200
```

Comment:
Why implement the methods here instead of in `series.py`?

Reply:
+1, the implementation seems to match the upstream modin version so it should be fine to leave in there.